src/components/Chat/InputMessageV2.tsx

```diff
@@ -1,13 +1,34 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
 import * as React from 'react';
-import {Alert, Image, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
-
+import {
+  Alert,
+  Image,
+  Linking,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View
+} from 'react-native';
+import ImagePicker from 'react-native-image-crop-picker';
+import DocumentPicker from 'react-native-document-picker';
+import {createThumbnail} from 'react-native-create-thumbnail';
 import ToastMessage from 'react-native-toast-message';
+
 import IconSend from '../../assets/icon/IconSendComment';
+import IconPlusAttachment from '../../assets/icon/IconPlusAttachment';
 import {colors} from '../../utils/colors';
 import dimen from '../../utils/dimen';
 import {normalizeFontSize} from '../../utils/fonts';
 import ToggleSwitch from '../ToggleSwitch';
-import SheetEmoji from './SheetEmoji';
+import BottomSheetAttachment from './BottomSheetAttachment';
+import BottomSheetGif from './BottomSheetGif';
+import {requestCameraPermission, requestExternalStoragePermission} from '../../utils/permission';
+import ImageUtils from '../../utils/image';
+import {ANONYMOUS, SIGNED} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code introduces several new imports and disables some eslint rules at the top of the file. While disabling eslint rules can sometimes be necessary, it's generally a good idea to avoid doing so unless absolutely necessary. In this case, the `no-await-in-loop` and `no-restricted-syntax` rules are being disabled twice, which is unnecessary. 

```diff
- /* eslint-disable no-await-in-loop */
- /* eslint-disable no-restricted-syntax */
- /* eslint-disable no-await-in-loop */
- /* eslint-disable no-restricted-syntax */
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->